dGame/dComponents/SkillComponent.h

```diff
@@ -190,6 +199,11 @@ class SkillComponent : public Component {
 	 */
 	uint32_t m_skillUid;
 
+	/**
+	 * Cache for looking up a behavior id via a skill ID
+	 */
+	static std::unordered_map<uint32_t, uint32_t> SkillBehaviorCache;
```

Comment:
Thoughts on making this camelCase for future readers?

Reply:
how about making it a proper private variable? :P